Source/Fuse.Common/Internal/iOSSystemFont.uno

```diff
@@ -329,6 +425,110 @@ namespace Fuse.Internal
 			throw new Exception("iOSSystemFont: No matching style in " + fileName);
 		}
 
+
+
+		internal static ObjC.Object GetMatchingSystemFontDescriptor(Fuse.SystemFont.Style fontStyle, Fuse.SystemFont.Weight fontWeight, Fuse.SystemFont.Design fontDesign, int fontSize)
+		{
+			string selectedFontStyle = "normal";
+			string selectedFontWeight = "normal";
+			string selectedFontDesign = "default";
+
+			switch(fontStyle) {
+				case Fuse.SystemFont.Style.Italic: selectedFontStyle = "italic";
+					break;
+			}
+
+			switch(fontWeight) {
+				case Fuse.SystemFont.Weight.UltraLight: selectedFontWeight = "ultralight";
+					break;
+				case Fuse.SystemFont.Weight.Thin: selectedFontWeight = "thin";
+					break;
+				case Fuse.SystemFont.Weight.Light: selectedFontWeight = "light";
+					break;
+				case Fuse.SystemFont.Weight.Normal: selectedFontWeight = "normal";
+					break;
+				case Fuse.SystemFont.Weight.Medium: selectedFontWeight = "medium";
+					break;
+				case Fuse.SystemFont.Weight.Semibold: selectedFontWeight = "semibold";
+					break;
+				case Fuse.SystemFont.Weight.Bold: selectedFontWeight = "bold";
+					break;
+				case Fuse.SystemFont.Weight.Heavy: selectedFontWeight = "heavy";
+					break;
+				case Fuse.SystemFont.Weight.Black: selectedFontWeight = "black";
+					break;
+			}
+
+			switch(fontDesign) {
+				case Fuse.SystemFont.Design.Default: selectedFontDesign = "default";
+					break;
+				case Fuse.SystemFont.Design.Monospaced: selectedFontDesign = "monospaced";
+					break;
+				case Fuse.SystemFont.Design.Rounded: selectedFontDesign = "rounded";
+					break;
+				case Fuse.SystemFont.Design.Serif: selectedFontDesign = "serif";
+					break;
+			}
+
+			return getSystemFontDesign(selectedFontStyle, selectedFontWeight, selectedFontDesign, fontSize);
+		}
+
+
+		[Foreign(Language.ObjC)]
+		static ObjC.Object getSystemFontDesign(string fontStyle, string fontWeight, string fontDesign, int fontSize)
+		@{
+			//weight
+			UIFontWeight selectedWeight;
+			if ([fontWeight isEqualToString:@"ultralight"]) {
```

Comment:
Code style nitpicking, but IMO all curly braces in this method should be removed (because of single-statement), or moved to their own lines.